source/getting-started/install.rst

```diff
@@ -77,6 +77,12 @@ Additional dependencies may be required depending on the host OS.
 
             macOS Docker Desktop resources requirement
 
+        .. tip::
+
+            **OrbStack** for **Mac** is supported by Exegol wrapper from ``v4.1.2``.
```

Comment:
Let's add an external link to OrbStack

Reply:
Do you have the official link to the project ?

Reply:
https://orbstack.dev/